subprojects/testing-jvm/src/integTest/groovy/org/gradle/testing/junit/JUnitIntegrationTest.groovy

```diff
@@ -146,29 +146,28 @@ class JUnitIntegrationTest extends JUnitMultiVersionIntegrationSpec {
 
     def canRunSingleTests() {
         when:
-        executer.withTasks('test').withArguments('-Dtest.single=Ok2').run()
+        succeeds("test", "--tests=Ok2*")
```

Comment:
❓Why do the pattern with `test.single` and `--tests` differs? (it has an extra wildcard characters)

Reply:
`test.single=Ok2` is implicitly interpreted as `**/Ok2*.class` which is about the same as `--tests=Ok2*`.  The wildcard is no longer implicit.